frontend/src/pages/HistoryPage.vue

```diff
@@ -1,27 +1,27 @@
 <template>
     <div class="row">
-        <div class="column visitedplaces-wrapper">
-            <div 
-                v-for="(visitedPlace,index) in visitedPlaces" 
-                :key="visitedPlace.id"
+        <div v-if="isPlacesLoaded" class="column visitedplaces-wrapper">
+            <div
+                v-for="(checkIn, index) in checkIns.byId"
```

Comment:
we made an agreement that you with byId also will create allIds properties as an array. It will look much better.